storybookjs/storybook
# Angular: Add "assets" build option

Closes #



## What I did


Added "assets" option to builder options, so assets can be configured in `angular.json` with Angular's format.

## How to test

1. Run a sandbox for template, e.g. `yarn task --task sandbox --start-from auto --template angular-cli/default-ts`
2. Add an asset. e.g. `assets/example_img.png`
3. Add "assets" to builder options in `angular.json`. e.g. `"storybook": { ..., "options": { ..., "assets": [ "src/assets" ] } }`
4. Use an asset in a story
5. Open Storybook in your browser
6. Check if a story can access the asset

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers

      make sure to add the `ci:merged` or `ci:daily` GH label to it.

`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
